sdk/identity/azure-identity-broker/src/main/java/com/azure/identity/broker/InteractiveBrowserBrokerCredentialBuilder.java

```diff
@@ -65,12 +65,11 @@ public InteractiveBrowserBrokerCredentialBuilder enableLegacyMsaPassthrough() {
      * Enables automatically using the signed-in user's account for brokered authentication instead of
      * of prompting the user with a login dialog.
      *
-     * @param useOperatingSystemAccount Boolean value to determine if the operating system account should be used.
-     * @return An updated instance of this builder with useOperatingSystemAccount set.
+     * @param useDefaultBrokerAccount Boolean value to determine if the operating system account should be used.
+     * @return An updated instance of this builder with useDefaultBrokerAccount set.
      */
-    public InteractiveBrowserCredentialBuilder useOperatingSystemAccount(boolean useOperatingSystemAccount) {
-        CredentialBuilderBaseHelper.getClientOptions(this).
-            setUseOperatingSystemAccount(useOperatingSystemAccount);
+    public InteractiveBrowserCredentialBuilder useDefaultBrokerAccount(boolean useDefaultBrokerAccount) {
```

Comment:
Since this API just a boolean switch with default value as false.
Can we just call it `useDefaultBrokerAccount()` or `enableDefaultBrokerAccount()` without the boolean parameter ?

@srnagar / @JonathanGiles

Reply:
happy to remove the boolean and go with `useDeafultBrokerAccount` (to match other languages so far, we'll settle on final name after archboard.)